tmt/export.py

```diff
@@ -60,6 +61,37 @@ def _nitrate_find_fmf_testcases(test):
             pass
 
 
+def markdown_to_html(text_field, test_md):
+    import markdown
+    try:
+        with open(test_md, 'r') as f:
+            text = f.read()
+            html = markdown.markdown(text)
+    except IOError:
+        raise ConvertError(f"Unable to open '{test_md}'.")
+
+    html_splitlines = html.splitlines()
+    res_str = str()
```

Comment:
Is this a result string? Could we call it `result` instead, to make the name obvious?